ui-library-call-readiness/src/App.tsx

```diff
@@ -0,0 +1,58 @@
+import { CallClientProvider, createStatefulCallClient, FluentThemeProvider } from '@azure/communication-react';
+import { initializeIcons, registerIcons } from '@fluentui/react';
+import {
+  DEFAULT_COMPONENT_ICONS
+} from '@azure/communication-react';
+import { useState } from 'react';
+import { PreCallChecksComponent } from './components/PreCallChecksComponent';
+import { PreparingYourSession } from './pages/PreparingYourSession';
+import { DeviceSetup } from './pages/DeviceSetup';
+import { TestComplete } from './pages/TestComplete';
+
+// Initializing and registering icons should only be done once per app.
+initializeIcons();
+registerIcons({ icons: DEFAULT_COMPONENT_ICONS });
+
+type TestingState = 'runningPreCallChecks' | 'deviceSetup' | 'finished';
+
+const USER_ID = 'user1'; // Replace with an Azure Communication Services User ID
```

Comment:
update comment to say for production apps they would retrieve a userId from their running service with a link to a server sample that can do this